assembly-combined-package/deploy-config/linkis-env.sh

```diff
@@ -139,3 +139,6 @@ LINKIS_VERSION=1.0.3
 
 # for install
 LINKIS_PUBLIC_MODULE=lib/linkis-commons/public-module
+
+## If SKYWALKING_AGENT_PATH is set, the Linkis components will be started with Skywalking agent
+SKYWALKING_AGENT_PATH=/appcom/config/skywalking-agent/skywalking-agent.jar
```

Comment:
Is it better to comment it out by default? It only opens if you need to open SkyWalking